### PR TITLE
Update script for adjusted outputs and modify row and col

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Description
Changes `ValidateContent` to use the updated format of validation results returned by running the `demisto-sdk` `validate` and `lint` command with the `json_output_path` argument. See this [PR](https://github.com/demisto/demisto-sdk/pull/1170) for more info.

## Before merging
- [ ] requires the `demisto/xsoar-tools` docker image is updated to use a version of the `demisto-sdk` that includes the changes from this [PR](https://github.com/demisto/demisto-sdk/pull/1170).

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] ~~Documentation~~ N/A

### CODE REVIEW

File: Packs/Base/Scripts/ValidateContent/ValidateContent.py [hunk 1]
```diff
@@ -109,6 +109,27 @@ def get_pack_name(zip_fp: str) -> str:
     return metadata.get('name', 'ServerSidePackValidationDefaultName')
 
 
+def adjust_linter_row_and_col(
+        error_output: Dict, row_offset: int = 2, row_start: int = 1, col_offset: int = 1, col_start: int = 0
+) -> None:
+    """Update the linter errors row and column numbering
+
+    Accounts for lines inserted during demisto-sdk extract, and that column numbering starts with one
```
Comment (Itay4) [inline Packs/Base/Scripts/ValidateContent/ValidateContent.py:117]: can you please add here why we're taking the max

File: Packs/Base/Scripts/ValidateContent/ValidateContent.py [hunk 2]
```diff
@@ -109,6 +109,27 @@ def get_pack_name(zip_fp: str) -> str:
...
+
+    Args:
+        error_output (Dict): A single validation result dictionary (validate and lint) from the total list
+        row_offset (int): The number of rows to adjust by
+        row_start (int): The lowest allowable number for rows
+        col_offset (int): The number of columns to adjust by
+        col_start (int): The lowest allowable number for columns
+    """
+    row, col = 'row', 'col'
+    if row in error_output:
```
Comment (Itay4) [inline Packs/Base/Scripts/ValidateContent/ValidateContent.py:127]: should we check that its a number to be on the safe side?

File: Packs/Base/Scripts/ValidateContent/ValidateContent.py [hunk 3]
```diff
@@ -325,14 +346,13 @@ def main():
...
-            for item in validation.values():
-                for error in item.get('outputs', []):
-                    if error.get('ui') or item.get('file-type') in {'py', 'ps1'}:
-                        outputs.append({
-                            'Name': item.get('display-name'),
-                            'Error': error.get('message'),
-                            'Line': error.get('line-number'),
-                        })
+            adjust_linter_row_and_col(validation)
+            if validation.get('ui') or validation.get('fileType') in {'py', 'ps1'}:
```
Comment (Itay4) [inline Packs/Base/Scripts/ValidateContent/ValidateContent.py:350]: i suppose the 2 loops that were here are no longer required cause of the changes in the sdk?

**Review by Itay4 — APPROVED** [review]

File: Packs/Base/Scripts/ValidateContent/ValidateContent.py (hunk 3 above)
Comment (avidan-H) [reply Packs/Base/Scripts/ValidateContent/ValidateContent.py:350]: yea exactly

File: Packs/Base/Scripts/ValidateContent/ValidateContent.py [hunk 4]
```diff
@@ -109,6 +124,51 @@ def get_pack_name(zip_fp: str) -> str:
...
+                        offset_for_file = code_fp_to_row_offset.get(filepath)
+                        if isinstance(offset_for_file, int):
+                            offset = offset_for_file
+                original_vector_value: Optional[Any] = error_output.get(vector)
+                if original_vector_value:
+                    error_output[vector] = str(max(int(original_vector_value) - offset, start))
+    except ValueError as e:
+        demisto.debug(f'Failed adjusting "{vector}" on validation result {error_output}'
+                      f'\n{e}')
+        raise e
```
Comment (Itay4) [inline Packs/Base/Scripts/ValidateContent/ValidateContent.py:169]: lets not stop the execution in case of failure

**Review by Itay4 — APPROVED** [review]

**Comment by avidan-H** [comment]
@Itay4 - fyi the changes won't work until we update the script's dockerimage with the latest changes to the `demisto-sdk`.

**Comment by Itay4** [comment]
> @Itay4 - fyi the changes won't work until we update the script's dockerimage with the latest changes to the `demisto-sdk`.

yea lets wait with merging this

**Comment by avidan-H** [comment]
Adding `docs-approved` since there is no new documentation that needs to be reviewed.

**Pull request merged**